### PR TITLE
Added BenEaterMachine

### PR DESCRIPTION
**Pull request by chelsea6502**
Hello!

I've been using a modified version of Symon as a debugging tool for my project where I'm porting BBC BASIC to a Ben Eater-style 6502: https://github.com/chelsea6502/BeebEater. It's become a valuable tool for me to debug serial IO, and I think it would be useful for others too. Hopefully it can make it into the mainline.

I'm not too familiar with Java, so let me know if I've missed anything important.

### CODE REVIEW

File: src/main/java/com/loomcom/symon/machines/BenEaterMachine.java [hunk 1]
```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright (c) 2016 Seth J. Morabito <web@loomcom.com>
+ *                    Maik Merten <maikmerten@googlemail.com>
```
Comment (sethm) [inline src/main/java/com/loomcom/symon/machines/BenEaterMachine.java:3]: Please set yourself as the copyright holder for this file, unless you'd prefer not to. It's your work, you deserve the credit!

**Review by sethm — COMMENTED** [review]
This looks like a good change to me, thanks very much for doing it! I left a comment about adding yourself as the copyright holder for the new file. Thanks!

File: src/main/java/com/loomcom/symon/machines/BenEaterMachine.java (hunk 1 above)
Comment (chelsea6502) [reply src/main/java/com/loomcom/symon/machines/BenEaterMachine.java:3]: Fair enough! I didn't even think that far 😂 I've just added my name

**Comment by chelsea6502** [comment]
Spotted a mistake in the README: the VIA and ACIA addresses were the wrong way around! I made another commit to fix this.

**Comment by sethm** [comment]
> Spotted a mistake in the README: the VIA and ACIA addresses were the wrong way around! I made another commit to fix this.

Thanks for spotting that! I didn't notice. These changes look good to me, so I'll go ahead and merge them!

**Comment by sethm** [comment]
This change has been merged into master! Thanks again for doing this.

**Pull request closed without merging**